cmd/mtc/main.go

```diff
@@ -376,12 +381,12 @@ func handleCaQueue(cc *cli.Context) error {
 }
 
 func handleNewAssertion(cc *cli.Context) error {
```

Comment:
Maybe we should rename this command and the corresponding inspect command.

Reply:
I was going back and forth on this since `assertion-request` is a little verbose, but that probably makes sense to avoid confusion.

Reply:
Today we use CSR. We can use AR? Or simply `new-request`?

Reply:
Oops, didn't see this before updating the PR. The current version has `assertion-request`, but we can tweak later. Some of the other CLI flags are pretty verbose too (`abridged-assertions`, `signed-validity-window`), so maybe we can just add short aliases (`aa`, `svw`, `ar`) if that seems ergonomic.